integrationtest/jms/src/test/java/org/hibernate/search/test/jms/master/JMSMasterTest.java

```diff
@@ -69,6 +73,10 @@
 	 */
 	private static final String CONNECTION_FACTORY_NAME = "java:/ConnectionFactory";
 
+	public static final Poller POLLER = Poller.milliseconds( 10_000, 100 );
+
+	private final QueryParser parser = new QueryParser( "id", TestConstants.stopAnalyzer );
```

Comment:
In this case it's fine, but generally speaking remember that such a `QueryParser` is not threadsafe. People will have to be careful when reusing helper methods from this class.